packages/server-core/src/projects/project-branches/project-branches.test.ts

```diff
@@ -70,12 +73,15 @@ describe('project-branches.test', () => {
 
     testUserApiKey = await app.service(userApiKeyPath).create({ userId: testUser.id })
 
-    await app.service(identityProviderPath).create(
+    await app.service(identityProviderPath)._create(
```

Comment:
```
await app.service(identityProviderPath)._create(identityProviderDataResolver.resolve(
      {
        type: 'github',
        token: `test-token-${Math.round(Math.random() * 1000)}`,
        userId: testUser.id
      }),
```
Instead of populating id, dates manually, you can call resolver using `identityProviderDataResolver.resolve` method to populate values.